db_client/alembic/versions/0040_add_metadata_field_to_family_document_.py

```diff
@@ -0,0 +1,34 @@
+"""Add valid_metadata field to family document model
+
+Revision ID: 0040
+Revises: 0039
+Create Date: 2024-07-01 17:34:26.909964
+
+"""
+
+import sqlalchemy as sa
+from alembic import op
+from sqlalchemy.dialects import postgresql
+
+# revision identifiers, used by Alembic.
+revision = "0040"
+down_revision = "0039"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.add_column(
+        "family_document",
+        sa.Column(
+            "valid_metadata", postgresql.JSONB(astext_type=sa.Text()), nullable=True
```

Comment:
What does the 'valid_' prefix add for the name? Any reason to not assume its valid an use 'metadata'?

Reply:
Yes! 'metadata' is a reserved field 
```sqlalchemy.exc.InvalidRequestError: Attribute name 'metadata' is reserved for the MetaData instance when using a declarative base class.```